.travis.yml

```diff
@@ -29,9 +39,22 @@ matrix:
       env: VIBED_DRIVER=libasync BUILD_EXAMPLE=0 RUN_TEST=0
   allow_failures:
     - d: dmd-beta
-    - d: ldc-1.1.0
+    - d: ldc-1.2.0-beta1
+      env: VIBED_DRIVER=libasync BUILD_EXAMPLE=0 RUN_TEST=0
+    - d: ldc-1.1.1
       env: VIBED_DRIVER=libasync BUILD_EXAMPLE=0 RUN_TEST=0
     - d: dmd-2.068.2
       env: VIBED_DRIVER=libasync BUILD_EXAMPLE=0 RUN_TEST=0
 
+before_install:
+  - pip3 install meson
```

Comment:
Well, technically >0.40, but pip will always get the latest version, so this is not an issue here.